h2integrate/resource/utilities/nrel_developer_api_keys.py

```diff
@@ -0,0 +1,185 @@
+import os
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+from h2integrate import ROOT_DIR
+
+
+developer_nrel_gov_key = ""
+developer_nrel_gov_email = ""
+
+
+def set_developer_nrel_gov_key(key: str):
+    """Set `key` as the global variable `developer_nrel_gov_key`.
+
+    Args:
+        key (str): API key for NREL Developer Network. Should be length 40.
+    """
+    global developer_nrel_gov_key
+    developer_nrel_gov_key = key
+
+
+def set_developer_nrel_gov_email(email: str):
+    """Set `email` as the global variable `developer_nrel_gov_email`.
+
+    Args:
+        email (str): email corresponding to the API key for NREL Developer Network.
+    """
+    global developer_nrel_gov_email
+    developer_nrel_gov_email = email
+
+
+def load_file_with_variables(fpath, variables=["NREL_API_KEY", "NREL_API_EMAIL"]):
+    """Load environment variables from a text file.
+
+    Args:
+        fpath (str | Path): filepath to a text file with the extension '.env' that
+            may contain the environment variable(s) in `variables`.
+        variables (list | str, optional): environment variable(s) to load from file.
+            Defaults to ["NREL_API_KEY", "NREL_API_EMAIL"].
+
+    Raises:
+        ValueError: If an environment variable is not found or found multiple times in the file.
+    """
+
+    # open the file and read the lines
+    with Path(fpath).open("r") as f:
+        lines = f.readlines()
+    if isinstance(variables, str):
+        variables = [variables]
+
+    # interate through each variable
+    for var in variables:
+        # find a line containing the environment variable
+        line_w_var = [line for line in lines if var in line]
+        if len(line_w_var) != 1:
+            raise ValueError(
+                f"{var} variable in found in {fpath} file {len(line_w_var)} times. "
+                "Please specify this variable once."
+            )
+        # grab the line containing the variable,
+        # assumes the line containing the variable is formatted as "variable=variable_value"
+        val = line_w_var[0].split(f"{var}=").strip()
+        # if var is NREL_API_KEY, set it as a global variable
+        if var == "NREL_API_KEY":
+            set_developer_nrel_gov_key(val)
+        # if var is NREL_API_EMAIL, set it as a global variable
+        if var == "NREL_API_EMAIL":
+            set_developer_nrel_gov_email(val)
+    return
+
+
+def set_nrel_key_dot_env(path=None):
```

Comment:
Nitpick: now that this is here, could you import it into the top-level `conftest.py` file instead of the HOPP one? If it does the same logic